src/oc/hoc.cpp

```diff
@@ -1400,7 +1400,6 @@ static void hoc_run1(void)	/* execute until EOF */
 	}
 	hoc_execerror_messages = 1;
 	if (pipeflag == 1) {	/*at this location multiple emacs errors */
-        hoc_pipeflush(); /* don't eat up stack space */
 	}else{
 		pipeflag=0;
 	}
```

Comment:
There is something stupid now here. Should be simplified more, right?

Reply:
IIRC this has ramifications into the HOC Interpreter parsing. We should review carefully.

Reply:
A grep for pipeflag in this PR source suggests that it is impossible for pipeflag to be 1.  The possibilities now are 0, 2, and 3. The no longer existing hoc_pipeflush() that was in oc/hocedit.cpp handled the situation where the emacs buffer execution generated a hoc error and we needed to cleanup and return to memacs to allow further editing.

Reply:
In that case we can drop the branching safely and just have 
```cpp
pipeflag=0;
```